spec/controllers/expenses_controller_spec.rb

```diff
@@ -189,4 +189,35 @@
       end
     end
   end
+
+  describe 'DELETE #destroy' do
+    let(:user) { create(:user) }
+    let(:expense) { create(:expense) }
+
+    context 'from login user and valid attributes' do
+      before do
+        sign_in user
+        delete :destroy,
+               params: {
+                 id: expense.id
+               }
+      end
+
+      it 'successful redirect after DELETE' do
+        expect(response).to redirect_to('/expenses')
+      end
+
+      it 'create flash message' do
+        expect(flash[:success]).to eq I18n.t('flash.controller.good_delete')
+      end
+
+      it 'has 302:Redirect' do
+        expect(response).to have_http_status(:redirect)
+      end
+
+      it 'save record to database with success' do
```

Comment:
It rather removes a record, doesn't it? :D

Reply:
Yes, yes XD